resize.py

```diff
@@ -1,11 +1,48 @@
 import cv2
+import time
+import numpy as np
+
  
```

Comment:
The functions in `resize.py` seem to be doing the same things as in `handler.py`. 
Which file is being used? Let's keep only one of them.

Reply:
resize.py is just to test the functions locally. But yes it can be removed from the repo

Reply:
If you want to keep it to test locally then name the file/module so that it's clear and also add a comment detailing this somewhere (possibly in the code & README)

Reply:
Done in the latest commit